packages/flutter/lib/src/physics/spring_simulation.dart

```diff
@@ -120,6 +120,9 @@ class SpringSimulation extends Simulation {
 
   @override
   String toString() => '$runtimeType(end: $_endPosition, $type)';
+
+  /// The end position of the simulation.
+  double get endPosition => _endPosition;
```

Comment:
nit: getters for private variables should be grouped with the private variable, see style guide.